src/App.js

```diff
@@ -1,5 +1,107 @@
+const MissionUtils = require('@woowacourse/mission-utils');
+
 class App {
-  play() {}
+  constructor() {}
+
+  getRandomNumber() {
+    this.computer = [];
+    while (this.computer.length < 3) {
+      const NUMBER = MissionUtils.Random.pickNumberInRange(1, 9);
+      if (!this.computer.includes(NUMBER)) {
+        this.computer.push(NUMBER);
+      }
+    }
+    MissionUtils.Console.print('숫자 야구 게임을 시작합니다.');
+  }
+
+  getInputNumber() {
+    MissionUtils.Console.readLine('숫자를 입력해주세요 : ', (number) => {
+      this.checkInputNumber(String(number).split(''));
```

Comment:
기본적으로 입력을 받을 때 `string` 타입으로 받더라구요! 그래서
```suggestion
      this.checkInputNumber(number.split(''));
```
이렇게 작성해주셔도 될 것 같습니다!